x11-wm/weston/Makefile

```diff
@@ -0,0 +1,114 @@
+# Created by: Johannes Lundberg <johalun0@gmail.com>
+# $FreeBSD$
+
+PORTNAME=		weston
+PORTVERSION=	3.0.0
```

Comment:
Please don't import this port, I'm not going to support it. I have my own fork of Weston that should be installed from git directly.

Reply:
OK done.